CRM/Contact/Import/Form/MapField.php

```diff
@@ -776,7 +778,7 @@ protected function saveMappingField($mapperKeys, array $saveMapping, string $cTy
         $saveMappingFields->website_type_id = $mapperKeys[$i][1] ?? NULL;
       }
       else {
-        if (CRM_Utils_Array::value('0', $mapperKeys[$i]) == 'phone') {
+        if (CRM_Utils_Array::value('0', $mapperKeys[$i]) == 'phone' || CRM_Utils_Array::value('0', $mapperKeys[$i]) == 'phone_ext') {
```

Comment:
We've been changing CRM_Utils_Array::value into native php as we come across it, to try to get rid of it, like:

`if (($mapperKeys[$i][0] ?? NULL) === 'phone' || ($mapperKeys[$i][0] ?? NULL) === 'phone_ext') {`

Reply:
Replaced by the suggestion
https://github.com/civicrm/civicrm-core/pull/20995/commits/3aca1423bad7c7dd9d2e0f9687b85823a2b64bb5